header.php

```diff
@@ -0,0 +1,24 @@
+<header>
+    <img id="logo_header" src="assets/img/logo-black.svg" alt="logo du site Blog-e-Boulga">
+
+    <ul id="list_nav_header">
+        <li><a href="index.php"><img src="assets/img/home.svg" alt="icône d'une maison" /> Accueil</a></li>
+        <li><a href="about_us.php"><img src="assets/img/buste.svg" alt="icône d'un buste d'homme" /> Qui sommes-nous ?</a></li>
+        <li><a href="#contactez_nous.php"><img src="assets/img/enveloppe.svg" alt="icône d'une enveloppe" /> Contactez-nous</a></li>
+    </ul>
+
+    <section id="burger_icon">
```

Comment:
wrap each img into a butto for better referencing and accessibility (and avoid part of the js)

Reply:
I have put it into an "a" instead, does it work rather than using a button?

```html
   <a id="aHeaderLogo" href="index.php">
        <img id="logo_header" src="assets/img/logo-black.svg" alt="logo du site Blog-e-Boulga">
    </a>
```

Reply:
Though I have used a button for my new burger icon

```html
    <button id="burger_icon" class="menuBurgerGlobal" aria-label="Main Menu">
        <svg width="50" height="50" viewBox="0 0 100 100">
            <path class="line line1" d="M 20,29.000046 H 80.000231 C 80.000231,29.000046 94.498839,28.817352 94.532987,66.711331 94.543142,77.980673 90.966081,81.670246 85.259173,81.668997 79.552261,81.667751 75.000211,74.999942 75.000211,74.999942 L 25.000021,25.000058" />
            <path class="line line2" d="M 20,50 H 80" />
            <path class="line line3" d="M 20,70.999954 H 80.000231 C 80.000231,70.999954 94.498839,71.182648 94.532987,33.288669 94.543142,22.019327 90.966081,18.329754 85.259173,18.331003 79.552261,18.332249 75.000211,25.000058 75.000211,25.000058 L 25.000021,74.999942" />
        </svg>
    </button>
```